pom.xml

```diff
@@ -244,7 +242,7 @@
         <jts.spatial4j.version>1.15.0</jts.spatial4j.version>
         <jts.version>1.14.0</jts.version>
         <jvnet-ogc.version>2.6.1</jvnet-ogc.version>
-        <karaf.version>4.2.1</karaf.version>
+        <karaf.version>4.2.2</karaf.version>
```

Comment:
- I don't believe the first bullet point here is needed. That `start.bat` file calls `ddf.bat` which calls `karaf.bat` which sets the `KARAF_LOG` environment variable.
- I didn't change the one for `ddf/distribution/docs/src/main/resources/content/_managing/_running/monitoring.adoc` because that'd make it so the users would have to know what `karaf.log` represented instead of `${home_directory}/data/log`
- I didn't change the `access.xml` one because that is similar but unrelated to the new Karaf system property.